src/ajax.js

```diff
@@ -15,64 +15,63 @@ const XHR_DONE = 4;
  */
 
 export function ajax(url, callback, data, options = {}) {
+  try {
+    let x;
+    let useXDomainRequest = false;
+    let method = options.method || (data ? 'POST' : 'GET');
```

Comment:
We should probably just drop this and let the caller specify explicitly. Ok for now tho.